sdk_extensions/zpages/src/main/java/io/opentelemetry/sdk/extensions/zpages/ZPageServer.java

```diff
@@ -59,7 +60,7 @@
  */
 @ThreadSafe
 public final class ZPageServer {
-  // The maximum number of queued incoming connections allowed on the HttpServer listening socket.
+  // The maximum number of queued incoming connections allowed on the HttpServer listening socket
```

Comment:
nit: this should end with `.`

Reply:
Should all comments end with `.`?

Reply:
I don't think we need to end them all with period, especially since this isn't actually a sentence. :)